tests/integration/questionnaire/test_questionnaire_last_viewed_guidance.py

```diff
@@ -0,0 +1,286 @@
+import random
```

Comment:
This file is almost identical to `test_questionnaire_last_viewed_guidance_hub.py`. Should this test file only have tests for the non-hub questionnaire and the hub test file have tests for the hub questionnaire?

Reply:
Something weird has happened here, as they are the same, I did write one for the hub, but somehow I have overridden it, fingers crossed my history is ok

Reply:
I see it whats happened, I was wondering how I got coverage. I copied the file, then decided to put it all in one, because it was sharing functionality, then didn't delete the copied file, doh.